Islandora-CLAW/islandora
# Remove duplicate media delete action

**GitHub Issue**: Resolves https://github.com/Islandora-CLAW/CLAW/issues/1212

# What does this Pull Request do?

Gets rid of our "Delete Media" action since there is now one in core.

# What's new?

- Deleted plugin
- Deleted schema entry for plugin
- Deleted exported config for action
- Deleted test for plugin
- Update hook to delete config if a user previously installed it

# How should this be tested?

- Confirm you have duplicate plugins by going to Admin > Content > Media and looking at the drop down.  There should be two 'Delete media' actions.
- Pull down this PR.
- Go to `localhost:8000/update.php` and run the update hook.
- Go back to Admin > Content > Media, and now you should only see one 'Delete media' action.

# Interested parties
@Islandora-CLAW/committers
